WORKLIST.txt

```diff
@@ -48,6 +49,10 @@ PLAYER:
 - equipped items are used to make attacks and to defend against attacks (raise AC, raise to-hit and damage stats, &c)
 - equippable items restricted to certain subset of all items ITEM_FLAG_EQUIPMENT, instead of hardcoded in CPlayer::Wield.
 - implement paginated inventory
+- how to remove cursed?
+- how to shoot bows
```

Comment:
This requires:
* creating bows
* creating weapons-that-need-ammo
* creating arrows
* maybe creating quivers
* distance attacks
* directional (NSEW) or targeted (requires "focus on monster") attacks

Reply:
Added an issue.